From d50d6b7b83d6eaa35de0a3ef7e5a085d9a224bca Mon Sep 17 00:00:00 2001
From: Michal Hrusecky <michal.hrusecky@nic.cz>
Date: Wed, 12 Dec 2018 15:02:36 +0100
Subject: [PATCH 2/6] mariadb: Drop default mariadb default file

Drop it as it is not needed and doesn't make much sense.

Signed-off-by: Michal Hrusecky <Michal@Hrusecky.net>
---
 utils/mariadb/Makefile            |  3 ---
 utils/mariadb/conf/mysqld.default | 10 ----------
 utils/mariadb/files/mysqld.init   |  9 ++-------
 3 files changed, 2 insertions(+), 20 deletions(-)
 delete mode 100644 utils/mariadb/conf/mysqld.default

diff --git a/utils/mariadb/Makefile b/utils/mariadb/Makefile
index 3032a1f..6057919 100644
--- a/utils/mariadb/Makefile
+++ b/utils/mariadb/Makefile
@@ -496,8 +496,6 @@ define Package/mariadb-server/install
 	$(INSTALL_BIN) files/mysqld.init $(1)/etc/init.d/mysqld
 	# DATA because spomebody thought giving CONF 0600 rights is a good idea
 	$(INSTALL_DATA) conf/my.cnf $(1)/etc/mysql
-	# DATA because spomebody thought giving CONF 0600 rights is a good idea
-	$(INSTALL_DATA) conf/mysqld.default $(1)/etc/default/mysqld
 	$(INSTALL_DIR) $(1)$(PLUGIN_DIR)
 	# DATA because spomebody thought giving CONF 0600 rights is a good idea
 	$(INSTALL_DATA) $(PKG_INSTALL_DIR)$(PLUGIN_DIR)/daemon_example.ini $(1)$(PLUGIN_DIR)
@@ -516,7 +514,6 @@ define Package/mariadb-server-extra/install
 endef
 
 define Package/mariadb-server/conffiles
-/etc/default/mysqld
 /etc/mysql/my.cnf
 $(PLUGIN_DIR)/daemon_example.ini
 endef
diff --git a/utils/mariadb/conf/mysqld.default b/utils/mariadb/conf/mysqld.default
deleted file mode 100644
index fe9c698..0000000
--- a/utils/mariadb/conf/mysqld.default
+++ /dev/null
@@ -1,10 +0,0 @@
-# The user and group that will run the MySQL server. The user mariadb is
-# automatically created by the mariadb-server package, hence that is the
-# default choice.
-
-#MY_USER=mariadb
-#MY_GROUP=mariadb
-
-# Additional arguments you want to pass to the MySQL server.
-
-#MY_ARGS=""
diff --git a/utils/mariadb/files/mysqld.init b/utils/mariadb/files/mysqld.init
index 5cdb9c6..1abbf9d 100644
--- a/utils/mariadb/files/mysqld.init
+++ b/utils/mariadb/files/mysqld.init
@@ -10,17 +10,12 @@ USE_PROCD=1
 
 MYSQLD=mysqld
 
-DEFAULT=/etc/default/$MYSQLD
 LOGGER="/usr/bin/logger -p user.err -s -t $MYSQLD"
 [ -x /usr/bin/logger ] || LOGGER=echo
 PROG=/usr/bin/$MYSQLD
 
-unset MY_ARGS MY_GROUP MY_USER
-
-[ -f $DEFAULT ] && . $DEFAULT
-
-my_user="${MY_USER:-mariadb}"
-my_group="${MY_GROUP:-mariadb}"
+my_user="mariadb"
+my_group="mariadb"
 
 mysql_get_var() {
 	/usr/bin/mysqld --help --verbose | sed -n 's|^'"$1"'[[:blank:]]\+||p'
-- 
2.19.2

